AnkiDroid/src/main/java/com/ichi2/libanki/utils/TimeUtils.kt

```diff
@@ -13,17 +13,13 @@
  You should have received a copy of the GNU General Public License along with
  this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package com.ichi2.libanki.utils
 
-package com.ichi2.libanki.utils;
+import java.text.SimpleDateFormat
+import java.util.*
 
-import java.text.SimpleDateFormat;
-import java.util.Locale;
-
-import androidx.annotation.NonNull;
-
-public class TimeUtils {
-    @NonNull
-    public static String getTimestamp(@NonNull Time time) {
-        return new SimpleDateFormat("yyyyMMddHHmmss", Locale.US).format(time.getCurrentDate());
+object TimeUtils {
+    fun getTimestamp(time: Time): String {
+        return SimpleDateFormat("yyyyMMddHHmmss", Locale.US).format(time.currentDate)
     }
```

Comment:
I am not good at Kotlin, yet. We are converting a `public static` method to Kotlin though, and I thought we needed to do JvmStatic and a companion object for java interoperability, like so:


```suggestion
   @JvmStatic
   companion object {
      fun getTimestamp(time: Time): String {
          return SimpleDateFormat("yyyyMMddHHmmss", Locale.US).format(time.currentDate)
      }
    }
```

But it did pass compilation and unit tests :thinking: 

I must be missing something!

Reply:
All the callers are Kotlin, so it's fine without the `@JvmStatic`

Reply:
Oh my - you mean we are far enough along in the conversion that for at least this case we don't need to worry about interop with Java :exploding_head: :laughing:. Awesome.
I guess that means there will be a big pruning of JvmStatic and companion objects later!
Thanks for explaining - obvious in hindsight but I guess I couldn't believe it.